skfp/preprocessing/input_output/aminoseq.py

```diff
@@ -0,0 +1,93 @@
+from collections.abc import Sequence
+from typing import Optional
+
+from rdkit.Chem import Mol, MolFromFASTA
+
+from skfp.bases import BasePreprocessor
+from skfp.utils.validators import check_strings
+
+
+class MolFromAminoseqTransformer(BasePreprocessor):
+    """
+    Creates RDKit ``Mol`` objects from amino-acid sequence strings.
+    This implementation uses sequences defined by FASTA [1]_ format.
+
+    Parameters
+    ----------
+    sanitize : bool, default=True
+        Whether to perform sanitization, i.e. basic validity checks, on created
+        molecules. For details see RDKit documentation [2]_.
+
+    flavor : int, default=1
+        Type of molecule [3]_.
+
```

Comment:
Why 2 empty lines? Also be slightly more specific, e.g. "see RDKit documentation [3]_ for details"